policyengine_us/variables/gov/states/az/hhs/tanf/az_hhs_tanf_earned_income_care_expense_disregard.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_us.model_api import *
+
+
+class az_hhs_tanf_earned_income_care_expense_disregard(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "Arizona Cash Assistance care expense earned income disregard"
+    definition_period = MONTH
+    defined_for = "az_hhs_tanf_eligible"
+
+    def formula(spm_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.az.hhs.tanf.eligibility.income.earned.care_expenses
+        # Get the age of the child(children) and the disabled adult(s)
+        person = spm_unit.members
+        age = person("age", period) * MONTHS_IN_YEAR
```

Comment:
Why multiply age by 12?

Reply:
We run into the issue that in monthly variables, the age is divided by 12 in the output.

Reply:
please file an issue in core. we need to be able to designate variables that don't get multiplied or divided depending on the period